History/HistoryManager/src/main/java/eu/neclab/ngsildbroker/historymanager/service/HistoryService.java

```diff
@@ -335,6 +337,11 @@ public void modifyAttribInstanceTemporalEntity(String entityId, String payload,
 							throw new ResponseException(ErrorType.InvalidRequest,
 									"instanceId in payload and in URL must be the same");
 						}
+					} else {
+						instanceIdAdd = jsonArray.get(0).getAsJsonObject().get(resolvedAttrId).getAsJsonArray().get(0).getAsJsonObject()
+								.get(NGSIConstants.NGSI_LD_INSTANCE_ID).getAsJsonArray().get(0).getAsJsonObject()
+								.get(NGSIConstants.JSON_LD_ID).getAsString();
+								jsonElement = setTemporalPropertyinstanceId(jsonElement, NGSIConstants.NGSI_LD_INSTANCE_ID, instanceIdAdd);
```

Comment:
code formatting is not proper, try to format it in eclipse only then commit the code.

Reply:
done